src/poliastro/tests/test_czml.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from numpy.testing import assert_allclose
+
+from poliastro.czml.extract_czml import CZMLExtractor
+from poliastro.examples import iss, molniya
+
+
+def test_czml_add_orbit():
+    start_epoch = iss.epoch
+    end_epoch = iss.epoch + molniya.period
+
+    Extractor = CZMLExtractor(start_epoch, end_epoch, 10)
```

Comment:
```suggestion
    extractor = CZMLExtractor(start_epoch, end_epoch, 10)
```

Let's use lowercase here, it was a bit confusing at the beginning (I'm so used to PEP8 naming that at first I thought it was a class)